docs/source/guide/qse-1-intro.md

```diff
@@ -0,0 +1,260 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# How do I use QSE?
+Here we show how to use QSE by means of a simple example.
+
+## Problem setup
+To use QSE, we call `qse.execute_with_qse()` with five “ingredients”:
+
+- A quantum circuit to prepare a state
+- A quantum computer or noisy simulator to return a QuantumResult from
+- An observable we wish to compute the error mitigated expectation value
+- A list of Check Operators which can be stabilizers, symmetries or anything else.
+- A Code Hamiltonian which defines the state with the least amount of errors
+
+## 1. Define a quantum circuit
+The quantum circuit can be specified as any quantum circuit supported by Mitiq.
+
+In the next cell we define (as an example) a quantum circuit that prepares the logical 0 state in the \[\[5,1,3\]\] code. For simplicity, we use Gram Schmidt orthogonalization to form a unitary matrix that we use as a gate to prepare our state.
```

Comment:
It looks to me like the escaping is not needed. This applies further down the file as well.
```suggestion
In the next cell we define (as an example) a quantum circuit that prepares the logical 0 state in the [[5,1,3]] code. For simplicity, we use Gram Schmidt orthogonalization to form a unitary matrix that we use as a gate to prepare our state.
```